lib/property.js

```diff
@@ -99,6 +99,27 @@ class Property {
     return this.visible;
   }
 
+  /**
+   * Sets the value and notifies the device if the value has changed.
+   * @returns true if the value has changed
+   */
+  setCachedValueAndNotify(value) {
+    const oldValue = this.value;
+    setCachedValue(value);
```

Comment:
Missing a `this.` before `setCachedValue`.

Reply:
Done